python/GafferSceneUI/SetEditor.py

```diff
@@ -58,11 +58,11 @@ def __init__( self, scriptNode, **kw ) :
 
 		searchFilter = _GafferSceneUI._SetEditor.SearchFilter()
 		emptySetFilter = _GafferSceneUI._SetEditor.EmptySetFilter()
-		emptySetFilter.userData()["UI"] = { "label" : "Hide Empty" }
+		emptySetFilter.userData()["UI"] = { "label" : "Hide Empty", "toolTip" : "Hide sets with no members" }
 		emptySetFilter.setEnabled( False )
 
 		emptySelectionFilter = _GafferSceneUI._SetEditor.EmptySetFilter( propertyName = "setPath:selectedMemberCount" )
-		emptySelectionFilter.userData()["UI"] = { "label" : "Hide Empty Selection" }
+		emptySelectionFilter.userData()["UI"] = { "label" : "Hide Empty Selection", "toolTip" : "Hide sets with no selected members" }
```

Comment:
I think we might also benefit from tooltips on the headers for the "Members" and "Selection" columns. That's a bit annoying to implement, because StandardPathColumn doesn't provide that functionality at present and it would be an ABI break to add a member for it. Maybe we do that for 1.4? Seems like the best way might be a `StandardPathColumn( const CellData &headerData, const InternedString property ... )` constructor, so you can specify icons and anything else you want?

Reply:
Great suggestion! Especially when coupled with the change in selection counting.  As mentioned above I've added these via new columns specific to the display of "Members" and "Selection", though providing a new constructor for StandardPathColumn would definitely be useful. I would have reached for that in my initial implementation if it were available...